docs/BR.md

```diff
@@ -1756,142 +1754,916 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+| Any other extension               | SHOULD NOT   | -            | __**TBD**__ |
+
+##### 7.1.2.1.2 Authority Key Identifier
+
+| __Field__                   | __Description__ |
+| ---                         | ------- |
+| `keyIdentifier`             | MUST be present. MUST be identical to the `subjectKeyIdentifier` field. |
+| `authorityCertIssuer`       | MUST NOT be present |
+| `authorityCertSerialNumber` | MUST NOT be present |
+
+##### 7.1.2.1.3 Basic Constraints
+
+| __Field__           | __Description__ |
+| ---                 | ------- |
+| `cA`                | MUST be set TRUE |
+| `pathLenConstraint` | SHOULD NOT be present |
+
+#### 7.1.2.2 Cross-Certified Subordinate CA Certificate Profile
+
+This Certificate Profile MAY be used when issuing a CA Certificate using the same Subject Name and Subject Public Key Information as an existing CA Certificate, whether a Root CA Certificate or Subordinate CA Certificate.
+
+Before issuing a CA Certificate using this Profile, the Issuing CA MUST confirm that the existing CA Certificate(s) are subject to these Baseline Requirements and were issued in compliance with the then-current version of the Baseline Requirements at time of issuance.
+
+__**TBD: Remarks about audits**__
+
+| __Field__                      | __Description__ |
+| ---                            | ------          |
+| `tbsCertificate`               | |
+| \ \ \ \ `version`              | MUST be v3(2) |
+| \ \ \ \ `serialNumber`         | MUST be a non-sequential number greater than zero (0) containing at least 64 bits of output from a CSPRNG. |
+| \ \ \ \ `signature`            | See [Section 7.1.3.2](#7132-signature-algorithmidentifier) |
+| \ \ \ \ `issuer`               | MUST be byte-for-byte identical to the `subject` field of the Issuing CA. See [Section 7.1.4.1](#7141-name-encoding) |
+| \ \ \ \ `validity`             | __**TBD**__ |
+| \ \ \ \ `subject`              | See [Section 7.1.2.2.1](#71221-cross-certified-subordinate-ca-naming) |
+| \ \ \ \ `subjectPublicKeyInfo` | See [Section 7.1.3.1](#7131-subjectpublickeyinfo) |
+| \ \ \ \ `issuerUniqueID`       | MUST NOT be present |
+| \ \ \ \ `subjectUniqueID`      | MUST NOT be present |
+| \ \ \ \ `extensions`           | See [Section 7.1.2.2.2](#71222-cross-certified-subordinate-ca-extensions) |
+| `signatureAlgorithm`           | Encoded value MUST be byte-for-byte identical to the `tbsCertificate.signature`. |
+| `signature`                    | |
+
+##### 7.1.2.2.1 Cross-Certified Subordinate CA Naming
+
+Provided that the Issuing CA has confirmed that the existing CA Certificate was issued in compliance with the then-current version of the Baseline Requirements, the Issuing CA MAY deviate from the requirements in [Section 7.1.4](#714-name-forms) as follows:
+
+The encoded `subject` name shall be byte-for-byte identical to the encoded `subject` name of the existing CA Certificate.
```

Comment:
To make sure I understand: You saw 7.1.4 as applying a MUST to all profiles, and then took the view that this MAY could not relax that MUST, is that... roughly correct? That's entirely my fault for wording this poorly.

The goal of "profiles" has been to try to replace "global variables", as it were, with local variables if they have different values. This is why 7.1.2.9.1 (used by the CA profiles) places a normative MUST on 7.1.4, and why 7.1.2.7.2/3/4/5 place the MUST on 7.1.4.

The problem with your proposed language is that it's not correct - it suggests that you can do a cross-certified sub-ca which is *not* byte-for-byte identical with the existing certificate, which we already forbid.

I _think_ the solution here is to actually remove all reference to Section 7.1.4, and just state byte for byte identical. Or are you worried 7.1.4 will still be read as a global?

Reply:
Thanks. This didn't really answer my questions directly, but I think your suggested text does seem to make it clear you see 7.1.4 applying globally.

I'm not a fan of the suggested language, but I'll leave this issue unresolved for now, and give a thought about how to clarify!

Reply:
Reworded in https://github.com/sleevi/cabforum-docs/pull/36/commits/35402abec99d0889783192db91ce34c08084f18d